containerd/containerd.go

```diff
@@ -67,6 +67,15 @@ var ContainerdCommand = cli.Command{
 			containerdDir = stateDir
 		}
 
+		defaultCpu := context.GlobalInt("default_cpus")
+		if defaultCpu <= 0 {
+			defaultCpu = 1
+		}
+		defaultMemory := context.GlobalInt("default_memory")
+		if defaultMemory <= 0 {
+			defaultMemory = 128
+		}
+
```

Comment:
Fixed. But I am wondering if there shouldn't be a package in `runv` collecting these types of hardcoded values (i.e. the default vcpu, memory limits?)